README.md

```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
 Important links:
 
 - [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+- https://slack.hackclub.io :: Join
+  [our Slack](https://starthackclub.slack.com/) community
+- Our public [chat logs](http://starthackclub.slackarchive.io/) if you want to
+  see what we're talking about
```

Comment:
I might make this a complete sentence.

Also, what are your thoughts on just leaving this in the topic of #introductions or #announcements? It feels a little out of place in at the top of our README (which could be thought of as our landing page).